app/services/concerns/xccdf/rule_group_relationships.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+module Xccdf
+  # Methods related to saving RuleGroupRelationships
+  module RuleGroupRelationships
+    extend ActiveSupport::Concern
+    included do
+      def save_rule_group_relationships
+        @op_rules_and_rule_groups = @op_rule_groups + @op_rules
+
+        ::RuleGroupRelationship.import!(
+          rule_group_relationships.select(&:new_record?), ignore: true
+        )
+
+        base = ::RuleGroupRelationship.left_outer_joins(dependent_rule_group: :benchmark)
+                                      .where(benchmark: { ref_id: @benchmark&.ref_id })
```

Comment:
@skateman After testing the `left_outer_joins` again with the `.where(benchmark: { ref_id: @benchmark&.ref_id })`, it does seem like it returns the right thing but you mentioned it might be limited to only rule_groups because it's joining with `dependent_rule_group`. I might be missing something or maybe I'm not testing correctly but it looks good to me. Let me know what you think!

Reply:
I don't think you need this, if you go with `::RuleGroupRelationship.all`, does it pass the tests without changes there?

Reply:
Yeah, it does pass with `::RuleGroupRelationship.all`